app/legacy_lib/insert_card.rb

```diff
@@ -37,17 +37,15 @@ def self.with_stripe(card_data, stripe_account_id=nil, event_id=nil, current_use
     # validate that the user is with the correct nonprofit
 
     card_data = card_data.slice(:holder_type, :holder_id, :stripe_card_id, :stripe_card_token, :name )
-    holder_types = {'Nonprofit' => :nonprofit, 'Supporter' => :supporter}
+    holder_types = { 'Supporter' => :supporter}
     holder_type = holder_types[card_data[:holder_type]]
     holder = nil
     begin
-      if holder_type == :nonprofit
-        holder = Nonprofit.select("id, email").includes(:cards).find(card_data[:holder_id])
-      elsif holder_type == :supporter
+      if holder_type == :supporter
         holder = Supporter.select("id, email, nonprofit_id").includes(:cards, :nonprofit).find(card_data[:holder_id])
       end
```

Comment:
Please remove if conditional because there's no other type of holder than :supporter going forward